palantir-java-format/src/main/java/com/palantir/javaformat/java/JavaInput.java

```diff
@@ -629,8 +629,8 @@ public void setCompilationUnit(JCCompilationUnit unit) {
         RangeSet<Integer> tokenRangeSet = TreeRangeSet.create();
         for (Range<Integer> characterRange0 : characterRanges) {
             Range<Integer> characterRange = characterRange0.canonical(DiscreteDomain.integers());
-            tokenRangeSet.add(characterRangeToTokenRange(
-                    characterRange.lowerEndpoint(), characterRange.upperEndpoint() - characterRange.lowerEndpoint()));
+            tokenRangeSet.add(characterRangeToTokenRange(characterRange.lowerEndpoint(), characterRange.upperEndpoint()
+                    - characterRange.lowerEndpoint()));
```

Comment:
I think having set BREAK_HERE on the levels containing binary operators might have been a bit of a mistake.

We only really want to inline things like the first string argument in a long string concatenation. We should probably limit it to just that.